tests/common/test_event.py

```diff
@@ -437,107 +439,214 @@ def test_elapsed_milliseconds(self):
-        extension_param_list_populated = [TelemetryEventParam('Name', "DummyExtension"),
-                                          TelemetryEventParam('Version', CURRENT_VERSION),
-                                          TelemetryEventParam('Operation', "DummyOperation"),
-                                          TelemetryEventParam('OperationSuccess', True),
-                                          TelemetryEventParam('Message', "TestMessage"),
-                                          TelemetryEventParam('Duration', 10), TelemetryEventParam('ExtensionType', ''),
-                                          TelemetryEventParam('OpcodeName', '')]
-        extension_param_list_with_defaults = EventLogger.add_default_parameters_to_event(extension_param_list_populated,
-                                                                                         set_values_for_agent=False)
-        self._assert_default_params_get_correctly_added(extension_param_list_with_defaults, default_parameters_expected)
-
-        parameters_expected = {"GAVersion": CURRENT_AGENT, 'ContainerId': "TEST_CONTAINER_ID", 'OpcodeName': "",
-                               'EventTid': 100, 'EventPid': 10, "TaskName": "", "KeywordName": ""}
-
-        # When some values are already populated in the TelemetryEventParamList.
-        extension_param_list_populated = [TelemetryEventParam('Name', "DummyExtension"),
-                                          TelemetryEventParam('Version', CURRENT_VERSION),
-                                          TelemetryEventParam('Operation', "DummyOperation"),
-                                          TelemetryEventParam('OperationSuccess', True),
-                                          TelemetryEventParam('Message', "TestMessage"),
-                                          TelemetryEventParam('Duration', 10),
-                                          TelemetryEventParam('ExtensionType', ''),
-                                          TelemetryEventParam('OpcodeName', ''),
-                                          TelemetryEventParam('EventTid', 100),
-                                          TelemetryEventParam('EventPid', 10)]
-        extension_param_list_with_defaults = EventLogger.add_default_parameters_to_event(extension_param_list_populated,
-                                                                                         set_values_for_agent=False)
-        self._assert_default_params_get_correctly_added(extension_param_list_with_defaults,
-                                                        parameters_expected)
-
-    @patch("threading.Thread.getName", return_value="HelloWorldTask")
-    @patch('os.getpid', return_value=42)
-    @patch("azurelinuxagent.common.event.get_container_id_from_env", return_value="TEST_CONTAINER_ID")
-    @patch("azurelinuxagent.common.event.datetime")
-    def test_add_default_parameters_to_agent_event(self, patch_datetime, *args):
```

Comment:
replaced by test_add_event_should_create_events_that_have_all_the_parameters_in_the_telemetry_schema (and its sibling tests for the other methods to create an event)